### ISSUE TITLE
Refered index.html ?

### ISSUE THREAD

**Issue by Thisisjuke** [JAremko/docker-x11-bridge#2]
@JAremko there is the URL : http://localhost:10000/index.html?encoding=png&password=<PASSWORD>

You are refering index.html, so I suppose I can set it somewhere, with shared volumes. Can I ?

**Comment by JAremko** [comment]
@Thisisjuke It should be served by `docker-x11-bridge` automatically

```
docker run -d \
 --name x11-bridge \
 -e MODE="tcp" \
 -e XPRA_HTML="yes" \
 -e DISPLAY=:14 \
 -e XPRA_PASSWORD=111 \
 -p 10000:10000 \   <-------------------------------  localhost:10000
 jare/x11-bridge

```

Unless you are using Docker Toolbox :thinking: 

Also You can try using http://xpra.org/html5/connect.html (but it may have different version and it will not be secure)

**Comment by Thisisjuke** [comment]
What I want to do is a custom HTML page, with my header, my footer etc.. Is this possible ? Or the page is doomed to be blank ? 
Thx to answer so quickly!

**Comment by JAremko** [comment]
@Thisisjuke  yes you can. We had something  ~similar~ exactly like that at http://spacemacs.org/

Here's the site part  https://github.com/syl20bnr/spacemacs.org/pull/28/commits/db6da0ac124e2b2edd56c2a5530f62d7172d4cf3 and the serve's one  https://github.com/JAremko/browsermax

unfortunately we had to disable it temporarily (see https://github.com/syl20bnr/spacemacs/issues/8634)
